homework/Kiet_Tran/lesson_1/sort_array_by_parity.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class Solution {
+
+    /**
+     * @param Integer[] $nums
+     * @return Integer[]
+     */
+    function sortArrayByParity(array $nums): array {
+        foreach ($nums as $index => $num) {
```

Comment:
what is the runtime of this approach? seem can be improved

Reply:
The runtime of this solution is O(n^2), but extra space is O(1)
It can be improved by other solution. We can use 2 for loops, one writes even number and one writes odd number
Its runtime is O(n) and extra space is O(n)
Thank you